distributions/aws/examples/rds-s3/README.md

```diff
@@ -63,35 +63,51 @@ aws s3 mb s3://$S3_BUCKET --region $CLUSTER_REGION
 
 Follow this [doc](https://www.kubeflow.org/docs/distributions/aws/customizing-aws/rds/#deploy-amazon-rds-mysql) to set up an AWS RDS instance. Please follow only section called `Deploy Amazon RDS MySQL`. 
```

Comment:
Can you add a line here saying this database instance will be used by Pipelines and Katib
and add a similar text in S3 step saying this bucket will be used to store artifacts from pipelines

asking because we are combing a few things now(e.g. no section for Katib) and it can be confusing for a user who does not know where/how these database and S3 bucket used in Kubeflow